core/src/main/java/org/apache/accumulo/core/client/admin/NewTableConfiguration.java

```diff
@@ -121,8 +128,22 @@ public NewTableConfiguration setProperties(Map<String,String> prop) {
    */
   public NewTableConfiguration enableSampling(SamplerConfiguration samplerConfiguration) {
     requireNonNull(samplerConfiguration);
-    SamplerConfigurationImpl.checkDisjoint(properties, samplerConfiguration);
-    this.samplerConfiguration = samplerConfiguration;
+    Map<String,String> tmp = new SamplerConfigurationImpl(samplerConfiguration).toTablePropertiesMap();
```

Comment:
Maybe a static utility to see if we can do an (unsafe) cast to `SamplerConfigurationImpl` to avoid an unnecessary object creation?